_includes/nav.html

```diff
@@ -8,7 +8,7 @@
         <span class="icon-bar"></span>
         <span class="icon-bar"></span>
       </button>
-      <a class="navbar-brand" href="{{ '/'  | prepend: site.baseurl }}"><img width="20" src="{{ '/assets/img/scala-js-logo.svg' | prepend: site.baseurl }}"> {{ site.title }}</a>
+      <a class="navbar-brand" href="{{ '/'  | prepend: site.baseurl }}"><img width="120" alt="Scala.js" src="{{ '/assets/img/scala-js-site-logo.svg' | prepend: site.baseurl }}"></a>
```

Comment:
Hum, so, if I understand this change correctly, this turns the top-left title *text* into an image. What motivated you to do this? To me, it seems this basically makes the user download two images instead of one.